example-config/data/tsoulis.face

```diff
@@ -1,15 +1,15 @@
 # Number of faces, boundary marker off
 12 0
 # Face index, nodes of face
-0 0 1 3
-1 1 2 3
-2 0 4 5
-3 0 5 1
-4 0 7 4
-5 0 3 7
-6 1 5 6
-7 1 6 2
-8 3 6 7
-9 2 6 3
-10 4 6 5
-11 4 7 6
+0   0	4	5
```

Comment:
out of curiosity, is this a problem in the tsoulis code?

Reply:
No, this is perfectly fine. During implementation, I had a bug leading to the miss conclusion that the input might be wrong - which wasn't the case. I reverted these files to the original ones.